### DISCUSSION TITLE
Machine Readable Prescription Drug File Negotiated Price Question

### DISCUSSION THREAD

**Discussion by AlysonMizunoKubo** [CMSgov/price-transparency-guide#98] [Q&A] [answered]
What negotiated price should be reported if there are multiple negotiated rates that apply under a given HIOS ID/EIN, Plan ID, Pharmacy, Drug?

**Comment by shaselton-usds** [comment, marked as answer]
Hi @AlysonMizunoKubo 

The `prices` attribute is an array to handle multiple negotiated rate: https://github.com/CMSgov/price-transparency-guide/blob/058865687f2b3e06043f53fb4bfbdd046d6ebaf5/schemas/prescription-drugs/prescription-drugs.json#L10

You can find the spirit of what you're asking here in this example: https://github.com/CMSgov/price-transparency-guide/blob/058865687f2b3e06043f53fb4bfbdd046d6ebaf5/examples/prescription-drugs/prescription-drugs.json#L14
